docs/tools/testing/dockerized-testing.md

```diff
@@ -7,6 +7,9 @@ head:
 
 # Dockerized setup
 
+Get started quickly with a Dockerized zkSync node using `npx zksync-cli dev start`! This tool simplifies local environment setups and other zkSync operations. Learn more and in the [zkSync CLI documentation](../zksync-cli/README.md).
```

Comment:
This start the `era-test-node` though? This part of the documentation would refer to running the Docker setup

Reply:
`npx zksync-cli dev start` would allow you to choose In-memory node (era-test-node) or Dockerized node